src/index.ts

```diff
@@ -1,65 +1,71 @@
-import { Command, flags } from "@oclif/command";
-import cli from "cli-ux";
+import { Args, Command, Flags, ux } from "@oclif/core";
 
 import * as parser from "./helpers/parser";
 import * as tsReader from "./helpers/tsReader";
 import * as paths from "./helpers/paths";
 import * as formatter from "./helpers/formatter";
 import * as generator from "./helpers/generator";
+import * as cli from "./helpers/cli";
 
 class MongooseTsgen extends Command {
+  static id = ".";
```

Comment:
What's this for?

Reply:
Without this, the help output is shown as

```
» ./bin/run --help
Generate a Typescript file containing Mongoose Schema typings.

USAGE
  $ mtgen mongoosetsgen MODEL_PATH [-c <value>] [-d] [-h] [-i <value>] [--no-format] [-o <value>] [-p <value>] [--debug] [--no-mongoose] [--no-populate-overload]
```

`oclif` infers the subcommand from the class name, which makes the help text misleading with that extra `mongoosetsgen` command. As described in the [recipe for Single Command CLI](https://oclif.io/docs/single_command_cli), we want to set the command name to "." to load an index.ts, so this simulates that. This is handled here: https://hub/oclif/core/blob/ff76400fb0bdfc4be0fa93056e86183b9205b323/src/help/docopts.ts#L80

And so, when we set the command id to `"."`, the help output is correctly 

```
» ./bin/run --help
Generate a Typescript file containing Mongoose Schema typings.

USAGE
  $ mtgen MODEL_PATH [-c <value>] [-d] [-h] [-i <value>] [--no-format] [-o <value>] [-p <value>] [--debug] [--no-mongoose] [--no-populate-overload]
```

Reply:
Great thanks!